README.md

```diff
@@ -108,6 +117,16 @@ Start CoreDNS and then query on that port (53). The query should be forwarded to
 response will be returned. Each query should also show up in the log which is printed on standard
 output.
 
+Sometimes port number 53 is occupied by system processes. In that case you can start the CoreDNS server
+while modifying the Corefile as given below so that the CoreDNS server starts on port 1053.
+
+~~~ corefile
+.:1053 {
+    forward . 8.8.8.8:53
+    log
+}
+~~~
+
```

Comment:
Move this segment to above, before the line that starts `If you have a Corefile without a port number specified it will,`